src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1087,6 +1087,19 @@ public override TypeInfo GetTypeInfo(SelectOrGroupClauseSyntax node, Cancellatio
         private void GetBoundNodes(CSharpSyntaxNode node, out CSharpSyntaxNode bindableNode, out BoundNode lowestBoundNode, out BoundNode highestBoundNode, out BoundNode boundParent)
         {
             bindableNode = this.GetBindableSyntaxNode(node);
+
+            // remove this check once we fix this issue
+            // https://github.com/dotnet/roslyn/issues/21180
+            if (!Root.FullSpan.Contains(bindableNode.FullSpan))
+            {
+                bindableNode = null;
+                lowestBoundNode = null;
+                highestBoundNode = null;
+                boundParent = null;
+                return;
+            }
+            ////
```

Comment:
we can't repro. see this - https://github.com/dotnet/roslyn/issues/21180

...

I added //// to show region that needs to be removed once 21180 is fixed.

// remove this check once we fix this issue
            // https://github.com/dotnet/roslyn/issues/21180
            if (!Root.FullSpan.Contains(bindableNode.FullSpan))
            {
                bindableNode = null;
                lowestBoundNode = null;
                highestBoundNode = null;
                boundParent = null;
                return;
            }
            ////